src/components/GetStartedPopover/GetStartedPopover.tsx

```diff
@@ -84,14 +85,16 @@ export const GetStartedPopover = ({}) => {
 
   return (
     <Flex className="split-button">
-      <Button
-        as="a"
-        href="/javascript/get-started"
-        size="large"
-        className="split-button__start"
+      <Link
+        href={{
+          pathname: '/[platform]/get-started',
+          query: { platform: DEFAULT_PLATFORM }
```

Comment:
Changed this to use next/link as well since I saw it was refreshing the whole page. I know this component is used in the home page and the overview page, but should I try to make it use the current platform if it exists or should we just leave it as the `default_platform` for now?